instrumentation/grpc-1.6/library/src/main/java/io/opentelemetry/instrumentation/grpc/v1_6/GrpcRpcAttributesGetter.java

```diff
@@ -37,4 +41,24 @@ public String method(GrpcRequest request) {
     }
     return fullMethodName.substring(slashIndex + 1);
   }
+
+  @Nullable
+  public List<String> metadataValue(GrpcRequest request, String key) {
+    if (request.getMetadata() == null) {
+      return null;
```

Comment:
Instead of `null`, consider returning `Collections.emptyList()` instead -- in general we recommend using empty/null-object instances for collections.

Reply:
Changed it